wire-runtime/src/jsMain/kotlin/com/squareup/wire/ProtoAdapter.kt

```diff
@@ -92,6 +92,11 @@ actual abstract class ProtoAdapter<E> actual constructor(
   /** Read a non-null value from `reader`. */
   actual abstract fun decode(reader: ProtoReader): E
 
+  /** Read a non-null value from `reader`. */
+  actual open fun decode(reader: ProtoReader32): E {
+    return decode(reader.asProtoReader())
+  }
```

Comment:
Is this codepath tested? I saw the new reader being tested and I saw the old reader being tested on a type with the new function. Didn't see the new reader being tested on a type which does not contain the new function, though.

Reply:
Ooooh I think it might be accidentally tested, but I think explicitly testing it is much better. Done.